### ISSUE TITLE
[bitnami/matomo] rm: cannot remove '/opt/bitnami/matomo': Permission denied exit code: 1

### ISSUE THREAD

**Issue by beertje44** [bitnami/containers#70872]
### Name and Version

bitnami/matomo:5.1.0

### What architecture are you using?

amd64

### What steps will reproduce the bug?

With or without a persistent /bitnami/matomo volume the container always exits with a permission denied to remove /opt/bitnami/matomo error. This is after the latest update from today.

### What is the expected behavior?

Container should start.

### What do you see instead?

Container enters bootloop.

**Comment by AlejandroAkbal** [comment]
Same, just updated and its crashing

```
2024-08-10T07:59:43.049414732Z ;5;6mmatomo ;5;5m07:59:43.04 ;5;2mINFO  ==> 
2024-08-10T07:59:43.050820871Z ;5;6mmatomo ;5;5m07:59:43.05 ;5;2mINFO  ==> Welcome to the Bitnami matomo container
2024-08-10T07:59:43.052466332Z ;5;6mmatomo ;5;5m07:59:43.05 ;5;2mINFO  ==> Subscribe to project updates by watching https://github.com/bitnami/containers
2024-08-10T07:59:43.053473746Z ;5;6mmatomo ;5;5m07:59:43.05 ;5;2mINFO  ==> Submit issues and feature requests at https://github.com/bitnami/containers/issues
2024-08-10T07:59:43.055288369Z ;5;6mmatomo ;5;5m07:59:43.05 ;5;2mINFO  ==> Upgrade to Tanzu Application Catalog for production environments to access custom-configured and pre-packaged software components. Gain enhanced features, including Software Bill of Materials (SBOM), CVE scan result reports, and VEX documents. To learn more, visit https://bitnami.com/enterprise
2024-08-10T07:59:43.055657654Z ;5;6mmatomo ;5;5m07:59:43.05 ;5;2mINFO  ==> 
2024-08-10T07:59:43.059577106Z ;5;6mmatomo ;5;5m07:59:43.05 ;5;2mINFO  ==> ** Starting Matomo setup **
2024-08-10T07:59:43.083609781Z ;5;6mmatomo ;5;5m07:59:43.08 ;5;2mINFO  ==> Generating sample certificates
2024-08-10T07:59:53.372071015Z Certificate request self-signature ok
2024-08-10T07:59:53.372108056Z subject=CN = example.com
2024-08-10T07:59:53.419956884Z realpath: /bitnami/apache/conf: No such file or directory
2024-08-10T07:59:53.421843629Z ;5;6mmatomo ;5;5m07:59:53.42 ;5;2mINFO  ==> Configuring Apache ServerTokens directive
2024-08-10T07:59:53.447200922Z ;5;6mmatomo ;5;5m07:59:53.44 ;5;2mINFO  ==> Configuring PHP options
2024-08-10T07:59:53.456159480Z ;5;6mmatomo ;5;5m07:59:53.45 ;5;2mINFO  ==> Setting PHP expose_php option
2024-08-10T07:59:53.470465708Z ;5;6mmatomo ;5;5m07:59:53.46 ;5;2mINFO  ==> Setting PHP output_buffering option
2024-08-10T07:59:53.495471676Z ;5;6mmatomo ;5;5m07:59:53.49 ;5;2mINFO  ==> Validating settings in MYSQL_CLIENT_* env vars
2024-08-10T07:59:53.606208770Z ;5;6mmatomo ;5;5m07:59:53.60 ;5;2mINFO  ==> Persisted Matomo installation detected
2024-08-10T07:59:53.609259090Z ;5;6mmatomo ;5;5m07:59:53.60 ;5;2mINFO  ==> Updating Matomo files in persisted data
2024-08-10T07:59:53.819173287Z ;5;6mmatomo ;5;5m07:59:53.81 ;5;2mINFO  ==> Restoring Matomo installation
2024-08-10T07:59:54.223830161Z rm: cannot remove '/opt/bitnami/matomo': Permission denied
```

**Comment by masterwishx** [comment]
it seems path changed to `/opt/bitnami/matomo` instead of `/bitnami/matomo`

**Comment by masterwishx** [comment]
sorry ,also not working

**Comment by masterwishx** [comment]
@javsalgar can it be fixed?

**Comment by javsalgar** [comment]
Hi,

Could you let us know from which version you are trying to install? We would like to reproduce the issue

**Comment by AlejandroAkbal** [comment]
Latest one available I think, definitely the one published the same day or the day before I posted my comment

**Comment by masterwishx** [comment]
> Hi,
> 
> Could you let us know from which version you are trying to install? We would like to reproduce the issue

I'm used latest tag so updated from version previous to latest, now goes back to version befor, working fine

**Comment by masterwishx** [comment]
Using in docker in Unraid x64 (Slackware 15)

**Comment by masterwishx** [comment]
`latest` `5.1.0-debian-12-r7` has issue, goes back to `5.1.0-debian-12-r6` fixed issue.

**Comment by juan131** [comment]
Hi everyone! Thanks so much for raising this bug.

We just released a new release (**5.1.0-debian-12-r8**) that fixes it, check the commit below:

- https://github.com/bitnami/containers/commit/c7f2ce7f1f0041ab03fed9c42f00dc95ae2a2a6f

Please give it a try and feel free to close the issue if you confirm the new image works for you.

Thanks in advance and sorry for the inconveniences.

**Comment by beertje44** [comment]
Works for me

**Comment by beertje44** [comment]
Please reopen if it doesn't for you

**Comment by masterwishx** [comment]
> Hi everyone! Thanks so much for raising this bug.
> 
> We just released a new release (**5.1.0-debian-12-r8**) that fixes it, check the commit below:
> 
> * [c7f2ce7](https://github.com/bitnami/containers/commit/c7f2ce7f1f0041ab03fed9c42f00dc95ae2a2a6f)
> 
> Please give it a try and feel free to close the issue if you confirm the new image works for you.
> 
> Thanks in advance and sorry for the inconveniences.

Thanks, Working fine
